plugins/search-react/api-report.md

```diff
@@ -118,6 +147,7 @@ export const SearchContextProvider: (
 // @public
 export type SearchContextProviderProps = PropsWithChildren<{
   initialState?: SearchContextState;
+  parentContext?: boolean;
```

Comment:
Also: I wonder if `parentContext` is a little unclear.  Maybe `inheritParentContext` or `useParentContext`...  Or maybe even `inheritParentContextIfAvailable` or `useParentContextIfAvailable`

Reply:
My first attempt was to use "useParentContext", but I renamed it so that it didn't sound like a hook name as Emma suggested.
I would like to be more explicit, so I would change the name to "inheritParentContextIfAvailable".
https://github.com/backstage/backstage/pull/13374#discussion_r957093515

Reply:
Done 🙌🏻